src/Event/SmsEvents.php

```diff
@@ -27,7 +27,7 @@
   const MESSAGE_GATEWAY = 'sms.message.gateway';
 
   /**
-   * Process and chunk a SMS message before it is queued, sent, or received.
+   * Process a SMS message before it is queued, sent, or received.
```

Comment:
> Event that is dispatched

Seems superfluous, because it is obviously an event being dispatched. Especially within context of the class?